tests/integration/sync/sync.spec.ts

```diff
@@ -868,7 +986,13 @@ describe('sync', function () {
           });
 
           expect(patchEntityResponse).toHaveStatus(StatusCodes.OK);
-          expect(patchEntityResponse.body).toMatchObject([baseSyncId]);
```

Comment:
Hey, Omer!
Thanks for stopping by. 
Please note, this is a test for sync. That is why, we do not check the validity of `patchEntity` response. It will be checked on different tests that specifically check patchEntity (such as in /integration/entity.spec.ts). 
Moreover, These requests are PRE-requests in order to check the validity of sync process. 
Please feel free to ask more questions on this matter. 
- Natalie and Niv😉